### PR TITLE
Add chart-operator 0.12.0 to WIP releases

### PR DESCRIPTION
**Pull request by rossf7**
- Adds to existing AWS WIP 11.0.2
- Creates new 11.2.1 Azure and KVM as there wasn't a WIP release.

### CODE REVIEW

File: app/changelog/chart-operator.yaml [hunk 1]
```diff
@@ -1,3 +1,13 @@
+- version: 0.12.0
+  description: Improvements to release and status resources. 
+  kind: changed
+  urls:
+    - https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0120-2020-03-09
```
Comment (rossf7) [inline app/changelog/chart-operator.yaml]: This is a minor release with quite a few changes. So this links to the chart-operator changelog.

File: app/changelog/chart-operator.yaml [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+- version: 0.11.5
```
Comment (rossf7) [inline app/changelog/chart-operator.yaml]: Adding missing changelog for 0.11.5.

**Comment by MartaD** [comment]
@giantswarm/team-halo  and @giantswarm/team-celestial are working on a release 11.2.1 for Azure (and kvm I think), shouldn't there already be a wip for it?

**Comment by rossf7** [comment]
Thanks @MartaD 

Azure 11.2.1 is defined in https://github.com/giantswarm/releases/pull/140 and KVM 11.2.1 in https://github.com/giantswarm/releases/pull/139

I'll check if we use those PRs and close this.

**Comment by rossf7** [comment]
I'll add this to the existing PRs and efforts. Thanks and sorry about the noise.

**Pull request closed without merging**